scripts/setup-adapters.sh

```diff
@@ -28,10 +31,33 @@ function install_aws-sdk-cpp {
   cmake_install -DCMAKE_BUILD_TYPE=Debug -DBUILD_SHARED_LIBS:BOOL=OFF -DMINIMIZE_SIZE:BOOL=ON -DENABLE_TESTING:BOOL=OFF -DBUILD_ONLY:STRING="s3;identity-management" -DCMAKE_INSTALL_PREFIX="${DEPENDENCY_DIR}/install"
 }
 
+function install_libhdfs3 {
```

Comment:
So I have an error here in the circle ci image. 
```
Find GoogleTest include path: /root/adapter-deps/hawq/depends/thirdparty/googletest/googletest/include
-- Find GoogleMock include path: /root/adapter-deps/hawq/depends/thirdparty/googletest/googlemock/include
-- Find Gtest library path: /root/adapter-deps/hawq/depends/thirdparty/googletest/build/googlemock/gtest/libgtest.a
-- Find Gmock library path: /root/adapter-deps/hawq/depends/thirdparty/googletest/build/googlemock/libgmock.a
-- Performing Test GoogleTest_CHECK_FINE
-- Performing Test GoogleTest_CHECK_FINE - Failed
```


It finds gtest but then fails on the check. I have downloaded the circle ci image and ran it in docker to reproduce it but still can't figure out why its failing. I will take another look at it tomorrow. 
As a sanity I even do a system wide gtest install on L51